include/s3select_functions.h

```diff
@@ -46,7 +46,9 @@ class s3select_functions : public __clt_allocator {
             {"extract", s3select_func_En_t::EXTRACT},
             {"dateadd", s3select_func_En_t::DATE_ADD},
             {"datediff", s3select_func_En_t::DATE_DIFF},
-            {"utcnow", s3select_func_En_t::UTCNOW}
+            {"utcnow", s3select_func_En_t::UTCNOW},
+            {"charlength", s3select_func_En_t::CHAR_LENGTH},
+            {"charlength", s3select_func_En_t::CHARACTER_LENGTH}
```

Comment:
the string parameter is the external name (user perspective)
the ENUM is internal (for same function , it need one enumeration)